README.md

```diff
@@ -89,22 +89,35 @@ endpoint of the tax calculator in order to provide the most coherent and
 reliable results. TaxJar support recommends using this endpoint for live
 calculations.
 
-## Development
+## Development Setup
```

Comment:
Chris recently linted the README to have hard-wrapped lines at 80 characters. Though it is technically inconsequential, my preference would be to continue doing that.